eg/Cli.py

```diff
@@ -149,7 +150,8 @@ class args:
                     if not NamedPipe.send_message('eg.app.Exit, ()'):
                         ctypes.windll.user32.MessageBoxA(
                             0,
-                            'EventGhost cannot restart - unknown error.\n',
+                            'EventGhost cannot restart.             \n\n'
+                            'Unknown Error.                         \n',
```

Comment:
One more question: will we get any exception or error number that we could give as addidtional information?